index.html

```diff
@@ -11,246 +11,142 @@
-                          <li><img src="images/Ellipse 1-2.png" alt="html-icon"></li>
-                          <li>HTML</li>
-                      </ul>
-                      <ul class="languages">
-                          <li><img src="images/Ellipse 1-3.png" alt="css-icon"></li>
-                          <li>CSS</li>
-                      </ul>
-                  </div>
-                  <div class="other">
-                    <ul class="framework">
-                      <li>Frameworks</li>
-                      <li>></li>
-                    </ul>
-                    <hr>
-                    <ul class="framework">
-                      <li>Skills</li>
-                      <li>></li>
-                    </ul>
-                    <hr>
-                  </div>
-              </div>
-            </div>
-          </div>
-      </div>
+
+  <section id="works">
+
   </section>
-  </main>
-  <footer>
-    <section id="contact" class="contact">
-      <div class="form-container">
-        <div class="contact-container">
-          <h1 class="contact-header">Contact Me</h1>
-          <p class="p-tag">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-        </div>
-        <div class="form-control">
-          <form action="https://formspree.io/f/mknebyrb" method="POST">
-            <input type="text" placeholder="Enter your fullname" name="name" maxlength="30" required>
-            <input type="email" name="email" id="email" placeholder="Input your email" required>
-            <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message" maxlength="500" required></textarea>
-            <div class="submit-button"><button type="submit" class="submit">Get in touch</button></div>
-          </form>
+
+  <section id="popup-overlay">
+    <div id="popup-content">
+      <div id="popup-title">
+        <h2></h2>
+        <div class="close-btn">&times;</div>
+      </div>
+      <p class="project-info">
+        <span class="project-company">CANOPY</span>
+        <span><img src="./images/Counter.png" alt="counter"></span>
+        <span class="project-type">Back End Dev</span>
+        <span><img src="./images/Counter.png" alt="counter"></span>
+        <span class="project-year">2015</span>
+      </p>
+      <img id="popup-img" src="./images/SnapshootPortfolio1.svg" alt="project screenshot">
+      <div id="popup-details">
+        <p id="popup-description"></p>
+        <div id="popup-skills-buttons">
+          <div id="popup-buttons">
+            <a class="enabled-btn live-link" target="_blank" rel="noopener" href="">See Live<i
+                class="fa-solid fa-arrow-up-right-from-square"></i></a>
+            <a class="enabled-btn source-link" target="_blank" rel="noopener" href="">See Source<i
+                class="fa-brands fa-github"></i></a>
+          </div>
```

Comment:
- [ ] Great job working on the Pop up I really appreciate your efforts but As I can see in the HTML you kept your HTML for the popup section the same but according to the requirements it is required to add dynamically like the works section so please remove all the hardcoded from the HTML and generate it using JS



![image](https://user-images.githubusercontent.com/58553711/197087763-a1283806-567b-450f-8e69-2b19b37684d0.png)

Reply:
Done